src/k8smanager.ts

```diff
@@ -188,36 +188,25 @@ export class K8sManager implements IK8sManager {
         const configPath = __dirname + path.sep + 'solutions/remotemonitoring/scripts/individual/deployment-configmap.yaml';
         const configMap = jsyaml.safeLoad(fs.readFileSync(configPath, 'UTF-8'));
         configMap.metadata.namespace = this._namespace;
-        configMap.data['security.auth.aad.endpoint.url'] = this._config.AzureActiveDirectoryEndpointUrl;
-        configMap.data['security.auth.tenant'] = this._config.AADTenantId;
-        configMap.data['security.auth.audience'] = this._config.ApplicationId;
+
+        configMap.data['keyvault.name'] = this._config.KeyVaultName;
```

Comment:
nit: I know it wasn't done before. Can we make alphabetical based on key names?

Reply:
Agreed however in the latest version https://github.com/Azure/pcs-cli/pull/484/files almost all of these are removed anyway aside from 3.